docs/source/API/core/policies/TeamVectorRange.rst

```diff
@@ -0,0 +1,93 @@
+``TeamVectorRange``
+===================
+
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
+.. code-block:: cpp
+
+    parallel_for(TeamVectorRange(team,range), [=] (int i) {...});
+    parallel_reduce(TeamVectorRange(team,begin,end),
+        [=] (int i, double& lsum) {...},sum);
+
+TeamVectorRange is a `nested execution policy <NestedPolicies.html>`_ used inside hierarchical parallelism. 
+In contrast to global policies, the public interface for nested policies is implemented 
+as functions, in order to enable implicit templating on the execution space type via 
+the team handle.
+
+Synopsis
+--------
+
+.. code-block:: cpp
+        
+    template<class TeamMemberType, class iType>
+    /* implementation defined */ TeamVectorRange(TeamMemberType team, iType count);
+    template<class TeamMemberType, class iType1, class iType2>
+    /* implementation defined */ TeamVectorRange(TeamMemberType team, iType1 begin, iType2 end);
+
+Description
+-----------
+
+.. cpp:function:: template<class TeamMemberType, class iType> TeamVectorRange(TeamMemberType team, iType count);
```

Comment:
you are omitting the return type , we cannot omit it even if it is specified in the synopsis 
if this is a problem for sphinx, just use the code block as before

Reply:
updated screenshots